scripts/kind-e2e/lib_operator_verify_subm.sh

```diff
@@ -245,6 +245,12 @@ function verify_subm_engine_pod() {
   kubectl get pod $subm_engine_pod_name --namespace=$subm_ns -o jsonpath='{.metadata.namespace}' | grep $subm_ns
 }
 
+function verify_subm_routeagent_daemonset() {
```

Comment:
Nice of you to add this but you're not calling it..

Reply:
lol :) woops

Reply:
done